packages/components/src/components/radio-group/radio-group.ts

```diff
@@ -204,12 +203,12 @@ export default class SdRadioGroup extends SolidElement implements SolidFormContr
     event.preventDefault();
   }
 
-  private handleLabelClick() {
+  /** Move focus to the checked radio (or the first one if none are checked) */
+  focus() {
```

Comment:
Shoelace used handleLabelClick, but this is now used in form.ts to focus when this element is the first invalid element in a form.  focus makes more sense, makes it more consistent, and makes it useable in the form method